sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/randomExpressions.scala

```diff
@@ -49,6 +49,8 @@ abstract class RDG extends LeafExpression with Nondeterministic {
   override def nullable: Boolean = false
 
   override def dataType: DataType = DoubleType
+
+  override def sql: Option[String] = Some(s"${prettyName.toUpperCase}($seed)")
```

Comment:
Just a note at here. Even if a user uses `rand()`, we will put a seed at here. (no need to make any change for this right now)